src/actions/commands/insert.ts

```diff
@@ -491,7 +540,7 @@ class CommandNavigateAutocompleteDown extends BaseCommand {
 @RegisterAction
 class CommandNavigateAutocompleteUp extends BaseCommand {
   modes = [ModeName.Insert];
-  keys = [['<C-p>'], ['<C-k>']];
+  keys = ['<C-p>'];
```

Comment:
This would be a breaking change. Not certain if this is ok...

Reply:
I couldn't get it to work otherwise, since the `<C-k>` shortcut will always prevent the `<C-k> <any> <any>` shortcut from firing.
I'm not even sure if `<C-k>` is actually referenced anywhere in the original vim for navigating up in the omnicomplete popup. All references to `<C-k>` in the neovim source code refer to digraphs.

Reply:
I'm trying to figure out what the appropriate keybinding would be for `CommandNavigateAutocompleteUp` and `CommandNavigateAutocompleteDown`.

These were initially added as a part of https://github.com/VSCodeVim/Vim/issues/1980. Likewise, I don't see `<c-j>` documented as autocompletedown in vim docs.